### ISSUE TITLE
Account: Consent: C4C - edit

### ISSUE THREAD

**Issue by MollieCourtenay** [healthlocker/healthlocker#120]
As a SLaM service user, I can view and update my C4C status, so that I can stay in control of whether researchers can contact me about research projects.

## Acceptance criteria

+ [ ] As a SLaM service user who is 18+ and on C4C register, I am can tick to leave the C4C register, then click to update my consent.
+ [ ] As a SLaM service user who is 18+ and on C4C register, I can edit my contact details preferences, and my choice about being contacted post-discharged.
+ [ ] As a SLaM service user who is 18+ who is not on C4C register, I can tick to express my interest in wanting to join the register, when I do this questions are revealed about contact post discharge and preferred contact details, once complete I click to ‘update’ my decision.

@LauraNevo please can you add most recent C4C wires in here, I think you did some updates but cant see them?

Linked to account epic: #28 
Linked to view C4C:  #129
Linked to clinician experience of C4C: #60

**Comment by MollieCourtenay** [comment]
@nelsonic - does this need splitting further?

**Comment by nelsonic** [comment]
Hi @MollieCourtenay the story is _almost_ there. 
The inclusion of the word "***and***" in on the first line suggests it's _two_ stories:
one for viewing the C4C status and the other for editing. 
The first thing we want to test is whether the SLaM service user will view their C4C status, 
`then` based on their feedback of _viewing_ their status/preferences we UX Test Team can _ask_ them_ "_what `else` would you **expect** to see on this screen_?"
(_without asking them a leading question like "do you want to edit this data...?" see what they expect._)

We _agree_ that _intuitively_ it can "_make sense_" to allow people to edit their account consent preferences, but very few people (_I'd bet fewer than 5% of users_) will ever edit/update so we should make _viewing_ one story and updating a _separate_ story with a lower priority in the backlog. 👍

**Comment by katbow** [comment]
@nelsonic agreed, this is shaping up nicely but just needs a bit more division to properly estimate 👍

**Comment by MollieCourtenay** [comment]
@nelsonic @katbow I will split the story. 
Just FYI - Consent for Contact (C4C) is a big research engagement programme that SLaM run - and being able to join and leave the network is an expected feature for HL.
We have plans (once the researcher element of HL is built and users are able to state their interest in a research study, that this would then prompt them to join the register - so there will be, in the future, more 'ways in' than it just sitting within Account.

**Comment by LauraNevo** [comment]
![c4c consent](https://cloud.githubusercontent.com/assets/3858131/23302118/4e5f5fe4-fa85-11e6-8db9-966023fdf1fb.png)
